api/app/src/mappings/fitbit/body.ts

```diff
@@ -56,8 +56,10 @@ export const mapToBody = (
   return body;
 };
 
-function checkSource(weight: string) {
-  return weight === "API" ? SourceType.manual : SourceType.device;
+function checkSource(weight_src: string) {
```

Comment:
let's keep our code consistent - we use "camelCase" not "snake_case" -> `weightSrc`

the only place we use "snake_cake" are our DAPI models, and now that I think about it, I'm not sure why

Reply:
Fixed